src/Sarif.Viewer.VisualStudio/CodeAnalysisResultManager.cs

```diff
@@ -744,9 +744,13 @@ internal bool TryResolveFilePathFromSolution(string solutionPath, string pathFro
                     return true;
                 }
             }
-            catch
+            catch (Exception ex) when (ex is ArgumentException ||
```

Comment:
>ArgumentException [](http://example.com/codeflow?start=45&length=17)

Generally, argument exception is not something you would swallow, as this indicates a condition a caller can check. There might not be a good helper to do so, though, so I don't know if my note is actionable.